index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>My Portfolio</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+
+</head>
+
+<body>
+    <header class="container">
+        <nav class="nav-item1">My Logo
+        </nav>
+        <nav class="nav-item2">
+        </nav>
+        <nav class="nav-item3">
+            <img src="images/toggle.png" alt="toggle">
+        </nav>
+    </header>
+    <main class="content">
+            <h1 class="box1">I’m Emmanuel
+                Glad to see you!</h1>
+        <p class="box2">
+            I'm a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h3 class="box3">
+            LET'S CONNECT
+        </h3>
+        <div class="box4">
+            <div class="social-media">
+               <a href="#">
+                 <img src="images/twitter.png" alt="twitter">
+                </a>
+            </div>
+            <div class="social-media">
+              <a href="#">
+                <img src="images/Linkedin.png" alt="Linkedln">
+              </a>
+            </div>
+            <div class="social-media">
+              <a href="#">
+                <img src="images/m.png" alt="m">
+              </a>
+            </div>
+            <div class="social-media">
+              <a href="#">
+                <img src="images/github.png" alt="github">
+              </a>
+            </div>
+            <div class="social-media">
+              <a href="#">
+                <img src="images/fingers.png" alt="two fingers">
+              </a>
+            </div>
+        </div>
```

Comment:
[OPTIONAL]
Following HTML & CSS best practices, it would be nice to use the `section` tag for the headline section. The portfolio will have many sections in the future. 😀